### PR TITLE
CI: Add markdown-link-check

### PR DESCRIPTION
**Pull request by MarkusAbtion**
This will add the GitHub Action for [Markdown Link Check](https://github.com/tcort/markdown-link-check).

I'm unsure about whether to run the Action on a schedule or for every push (or both):
Running it on a schedule will allow us to catch links that broke by themselves.
Running it on push will allow us to catch broken links that we just added.

Thoughts?

The `mlc_config.json` file is set to:
- Add headers to GitHub Docs links.
GitHub throws a 403 error, when headers aren't added
- Ignore internal Asana links
- Ignore the gitmessage repo
- Ignore LinkedIn links
LinkedIn throws a [status 999](https://wiert.me/2019/10/23/when-linkedin-throws-http-result-code-999-at-you-it-means-it-does-not-like-you) to avoid anyone from requesting the site programatically
- Ignore the abtion.io website
Because it's password-protected

### CODE REVIEW

**Comment by hcarreras** [comment]
Awesome addition! 
I agree with your analysis. Should we do it for both cron jobs + PRs? 
We do not want anything committed that is broken.

File: .github/workflows/markdown-link-check.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  # Triggers the workflow every Monday at 9AM
+  schedule:
+    - cron: "0 9 * * 1"
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
```
Comment (runephilosof-abtion) [inline .github/workflows/markdown-link-check.yml:14]: missing eol at eof

**Review by runephilosof-abtion — APPROVED** [review]
as Hari mentions it would be good to also trigger on PRs

**Comment by MarkusAbtion** [comment]
Update on this:
I've been spending a hot minute trying to figure out how to handle pages protected by Cloudfront. 
It's not possible with this tool, which means that we'll have to add them to the ignore list.

Therefore, I'm thinking of closing this PR until we've found a better tool or the author has updated it to use puppeteer.

**Pull request closed without merging**